TUM Campus App/TumDataManager.swift

```diff
@@ -77,9 +77,10 @@ class TumDataManager {
         }
     }
     
-    func search(_ receiver: TumDataReceiver, query: String) {
+    func search(_ receiver: TumDataReceiver, query: String, searchIn managersToSearchIn:[TumDataItems]? = nil) {
```

Comment:
Wanted to do so, but Xcode gave an error that it's not possible to use instance variables as default parameters.